syntaxes/fsh.tmLanguage.json

```diff
@@ -26,15 +26,15 @@
 			"patterns": [
 				{
 					"name": "keyword.control.fsh",
-					"match": "\\b(Profile|Extension|SimpleElement|ComplexElement|Parent|Mixin|Mixins|Id|Title|Description|Datatype|Property|Element|Abstract|Entry|Group|Rules|Instance|InstanceOf|Invariant|Expression|Severity|XPath|Mapping|Source|Target|Metadata|Slice|ValueSet|Language|Alias|Usage)\\b"
+					"match": "\\b(Alias|CodeSystem|Expression|Extension|Description|Id|Instance|InstanceOf|Invariant|Mapping|Mixins|Parent|Profile|RuleSet|Severity|Source|Target|Title|Usage|ValueSet|XPath)(?=:)\\b"
```

Comment:
I think it is better to change the lookahead to allow any number of whitespace before the `:`, since that is what is allowed in the grammar. I can't test for sure because I don't know how to use this branch of the extension locally (PS I should learn to do that), but I'm pretty sure as it is now:
```
Profile: MyProf
```
Will highlight `Profile`, but 
```
Profile  :  MyProf
```
Will not highlight.

Reply:
Ah yes I believe you're correct and I wasn't thinking of this case! I'll update!